java/build.sh

```diff
@@ -3,12 +3,24 @@ GROUP_ID="com.nvidia.cuvs"
 SO_FILE_PATH="./internal"
 
 if [ -z "$CMAKE_PREFIX_PATH" ]; then
-    export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
+  export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
 fi
 
 cd internal && cmake . && cmake --build . \
-  && cd .. \
-  && mvn install:install-file -DgroupId=$GROUP_ID -DartifactId=cuvs-java-internal -Dversion=$VERSION -Dpackaging=so -Dfile=$SO_FILE_PATH/libcuvs_java.so \
+  && cd ..
+
+# Generate Panama FFM API bindings and update (if any of them changed)
+/bin/bash panama-bindings/generate-bindings.sh
```

Comment:
```suggestion
./panama-bindings/generate-bindings.sh
```

`panama-binding/generate-bindings.sh` already has `/bin/bash` in its shebang... let's please not duplicate that here.

If you haven't already, this change should be accompanied with making `generate-bindings.sh` executable.

```shell
chmod +x ./panama-bindings/generate-bindings.sh
```